docs/docs/README.md

```diff
@@ -0,0 +1,581 @@
+# Welcome to WorkFlow Launcher
+
+## Overview
+
+[WorkFlow Launcher (WFL)](https://github.com/broadinstitute/wfl.git) 
+is a workload manager.
+
+It runs as you, with your credentails, from your laptop, and
+communicates with other services as necessary to manage a workload.
+
+It can also be deployed to run as a service in the cloud.
+
+For more on Workflow Launcher's role in the Terra infrastructure see 
+[Workflow Launcher's role in Terra (./docs/docs/terra.md)](./terra.md).
```

Comment:
Why does this doc tell the user to refer to terra here and then below describes it's role anyway? Should it instead link to either the area below or delete the area below and keep the role this plays in terra?

Reply:
Good question, I guess when we factored out the role stuff to a separate file, we didn't update this readme. I will delete the below section.